BUILD.md

```diff
@@ -65,9 +65,7 @@ See [xroad-catalog-persistence/README.md](xroad-catalog-persistence/README.md#bu
 Profiles can be used to configure different features in X-Road Catalog. By default, X-Road Catalog supports 1 profile, 
 the `default` profile with all configurations in place.
```

Comment:
I think since we only do have the one profile at the moment, this could probably be removed altogether.